### PR TITLE
Add Google RTC Audio Processing with RTNR to MT8195 topology

### PR DESCRIPTION
**Pull request by MingJenTai**
This PR adds Google RTC Audio Processing with RTNR to MT8195
topology.

### CODE REVIEW

**Review by lgirdwood — APPROVED** [review]

File: tools/topology/topology1/sof/pipe-google-rtc-audio-processing-rtnr-capture.m4 [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+include(`bytecontrol.m4')
+include(`enumcontrol.m4')
+include(`google_rtc_audio_processing.m4')
+include(`rtnr.m4')
+
+#
+# Controls
+#
+
+include(`google_rtc_audio_processing_default.m4')
```
Comment (johnylin76) [inline tools/topology/topology1/sof/pipe-google-rtc-audio-processing-rtnr-capture.m4:24]: This file is missing.

**Comment by lgirdwood** [comment]
@kuanhsuncheng can MTK pls review.

**Comment by johnylin76** [comment]
Look good to me.
Although there are small updates in google-rtc-audio recently, I will help with that with another PR. Thanks

**Review by johnylin76 — APPROVED** [review]

File: tools/topology/topology1/sof-mt8195-mt6359-rt1019-rt5682.m4 [hunk 2]
```diff
@@ -53,9 +90,13 @@ PIPELINE_PCM_ADD(
 # Low Latency capture pipeline 3 on PCM 18 using max 2 channels of s16le
 # Set 2000us deadline with priority 0 on core 0
 PIPELINE_PCM_ADD(
-	ifdef(`RTNR', sof/pipe-rtnr-capture.m4, sof/pipe-passthrough-capture.m4),
+	ifdef(`RTNR',
+        ifdef(`GOOGLE_RTC_AUDIO_PROCESSING',
+        sof/pipe-google-rtc-audio-processing-rtnr-capture.m4,
+        sof/pipe-rtnr-capture.m4),
+    sof/pipe-passthrough-capture.m4),
```
Comment (yaochunhung) [inline tools/topology/topology1/sof-mt8195-mt6359-rt1019-rt5682.m4:97]: please align. thanks

**Review by yaochunhung — COMMENTED** [review]
Suggest to change commit message title as 
topology1: mt8195: support Google rtc audio process with RTNR

File: tools/topology/topology1/sof-mt8195-mt6359-rt1019-rt5682.m4 (hunk 2 above)
Comment (MingJenTai) [reply tools/topology/topology1/sof-mt8195-mt6359-rt1019-rt5682.m4:97]: Updated. Thanks.

**Comment by MingJenTai** [comment]
> Suggest to change commit message title as topology1: mt8195: support Google rtc audio process with RTNR

Updated. Thanks.

**Review by yaochunhung — APPROVED** [review]

**Review by cujomalainey — APPROVED** [review]

**Comment by cujomalainey** [comment]
tools build is fine, none of the code here affects intel so I am merging and not waiting for Intel CI to finish.

**Pull request merged**